libs/vm_monitor/src/bin/monitor.rs

```diff
@@ -0,0 +1,26 @@
+#[cfg(target_os = "linux")]
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
```

Comment:
This file should have a (maybe brief) comment explaining why the monitor is available both embedded inside compute_ctl _and_ as a standalone binary